dbt_style_guide.md

```diff
@@ -230,6 +240,48 @@ models:
               field: id
 ```
 
+## Metrics style guide
+
+* dbt Metrics definitions are a special kind of YAML file, with additional conventions.
+* Metrics names must begin with a letter, cannot contain whitespace, and should be all lowercase.
+* Tags and/or Meta properties should be used to organize metrics at the business function level.
+* Meta properties should be used to track metric definition ownership.
+* The [minimum required properties](https://docs.getdbt.com/docs/building-a-dbt-project/metrics#available-properties) must be present in the metric definition.
+* For up-to-date information on metrics, please see the [metrics docs on defining a metric](https://docs.getdbt.com/docs/building-a-dbt-project/metrics#defining-a-metric) or the [dbt-labs/metrics README](https://github.com/dbt-labs/dbt_metrics#readme)
+
+
+### Example Metric YAML
+```yaml

```

Comment:
Should we include an example of a `base__` metric?

Reply:
I can include it in the structure! I plan on refactoring some derived metrics to demonstrate the base__ prefix in our project, if this convention is approved